### DISCUSSION TITLE
[QnA] 5주차 (~ 03.13)

### DISCUSSION THREAD

**Discussion by juunzzi** [woowacourse-study/2022-no-posting-you-die#3] [Q&A]
ㅎㅇ

**Comment by juunzzi** [comment]
@byhhh2 

- 의존성을 주입한다
- 의존성이 역전되다

이 두 문장을 설명해주실수있나요 ㅠㅠ? 무비의 글을 보고 어떤 느낌인지..? 는 조금 감이오는데 명확히 설명이안되네요 머릿속에서

**Reply by byhhh2** [reply]
의존성을 주입한다는 말은 Bundle이라는 클래스가 있다면 생성자 인자에 LottoModel을 넣어주면서 lotto에 대한 의존성을 주입해준다는 말같아요! 

또 역전됐다는 말은 보통 Lotto가 도메인이면 LottoBundle이라는  구체적인 클래스에 의존하잖아요? 근데 DI는 Bundle(상위 모듈)에 의존하기 때문에 역전됐다는 말을 쓰는 것 같아요 🤔

**Reply by juunzzi** [reply]
오 조금 더 명확해진 것 같아요 ㅋㅋ 역전되었다 이 말이 안와닿앗는데, 조금 더 내게 다가온 느낌.. ㅋㅋ 감사합니다 무비비빅~~
